source/docs/software/wpilib-tools/robotbuilder/writing-code/robotbuilder-writing-command-code.rst

```diff
@@ -1,90 +1,137 @@
 Writing the Code for a Command
 ==============================
 
-.. important:: RobotBuilder has been updated to support the new commandbased framework! Unfortunately, this documentation is outdated and only for the old commandbased framework. Individuals interested in updating this documentation can open a pull request on the `frc-docs <https://github.com/wpilibsuite/frc-docs>`__ repository.
-
 Subsystem classes get the mechanisms on your robot moving, but to get it to stop at the right time and sequence through more complex operations you write Commands. Previously in :doc:`writing the code for a subsystem <robotbuilder-writing-subsystem-code>` we developed the code for the Claw subsystem on a robot to start the claw opening, closing, or to stop moving. Now we will write the code for a command that will actually run the Claw motor for the right time to get the claw to open and close. Our claw example is a very simple mechanism where we run the motor for 1 second to open it or 0.9 seconds to close it.
 
 Closeclaw Command in RobotBuilder
 ----------------------------------
 
 .. image:: images/writing-command-code-1.png
 
-This is the definition of the Closeclaw command in RobotBuilder. Notice that it requires the Claw subsystem. This is explained in the next step.
+This is the definition of the CloseClaw command in RobotBuilder. Notice that it requires the Claw subsystem. This is explained in the next step.
```

Comment:
I feel like the commands need some sort of emphasis, perhaps *italicized* or in a ``inline code block``.